spacy/cli/debug_data.py

```diff
@@ -686,14 +756,28 @@ def _format_labels(
     return ", ".join([f"'{l}'" for l in cast(Iterable[str], labels)])
 
 
-def _get_examples_without_label(data: Sequence[Example], label: str) -> int:
+def _get_examples_without_label(
+    data: Sequence[Example],
+    label: str,
+    component: str = "ner",
```

Comment:
If you don't provide "ner" or "spancat" this method is going to crash, I think. I think `Literal` types would be enough to make it clear, but there could be other alternatives / error handling.

Reply:
Updated to `Literal` type in this commit: https://github.com/explosion/spaCy/pull/10070/commits/0c289b3faca192c604fd403b50c4ebca3534d49a

I was tempted to do the following:

```python
count = 0
if component not in ["ner", "spancat"]:
    return count
else:
   for eg in data:
       ...
```

But the semantics of returning a count of zero and silently passing an "error" may not be ideal. So I opted for just typing it as a `Literal`.

Reply:
It wouldn't be a problem if this particular internal method threw the right kind of error, but an undefined local variable error is going to confuse someone in the future unnecessarily.

And we wouldn't want to just use `Literal` for user-facing methods, but I think this is good enough for this private method right now.